examples/monitors/multiple_monitors_with_shared_payload_template/payload-template

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
let's provide example templates for all types of monitors we support with all fields populated

Reply:
Provided different template files for each monitor type in here 55fae3c. Yet nearly all the monitor types have the same payload schema, so I've changed the values of the payload items.

Reply:
thanks for adding new payload items. they look like they are pattern monitor payloads. I think we can change the folder structure under examples/monitors folder to match the type of alert_defs (monitors) we have.
examples/monitors/
  -> pattern-check/
  -> pattern-skyline/
  -> correlated-signal/
  -> metric-alert/

In each folder a *.tf and a corresonding payload.json file which has the json payload of corresponding alert type. you can find the schema of the payload jsons for each alert type in core/type.go. e.g. MetricJobPayload, PatternCheckJobPayload, PatternSkylineJobPayload, CorrelatedSignalJobPaylod.
Using the admin UI you can create one of each monitor and check the payload from the network panel which can be provided here as documentation with some modification.

Reply:
Separated the example configs and created different example payloads for each corresponding alert type in commits 4a9e866 and c83d5eb. Also added an extra use-case example for creating multiple resources with the same payload but the scope (tag), in `examples/monitors/with-payload-template`.